Release v0.1.0-alpha (#76)
open-telemetry/opentelemetry-go-instrumentation: CHANGELOG.md

# Changelog

> All notable changes to this project will be documented in this file.

Reviewer: very nit: I find that ["project"](https://dictionary.cambridge.org/dictionary/english/project) term is not properly used. I propose to be more precise 😉 

Moreover, I do not understand why not using Present Simple tense.

```suggestion
All notable changes to OpenTelemetry Go Automatic Instrumentation
is documented in this file.
```

PS. I am aware that this copied from https://keepachangelog.com/en/1.1.0/ 😉

Author: ```suggestion
All notable changes to OpenTelemetry Go Automatic Instrumentation are documented in this file.
```

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).

> This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

Reviewer: nit:

```suggestion
OpenTelemetry Go Automatic Instrumentation adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
```

## [Unreleased]

## [v0.1.0-alpha] - 2023-04-13

This is the first release of OpenTelemetry Go Automatic Instrumentation.

> [Unreleased]: https://github.com/open-telemetry/opentelemetry-go/compare/v0.1.0-alpha...HEAD

Reviewer 2: ```suggestion
[Unreleased]: https://github.com/open-telemetry/opentelemetry-go-instrumentation/compare/v0.1.0-alpha...HEAD
```

> [v0.1.0-alpha]: https://github.com/open-telemetry/opentelemetry-go/releases/tag/v0.1.0-alpha

Reviewer 2: ```suggestion
[v0.1.0-alpha]: https://github.com/open-telemetry/opentelemetry-go-instrumentation/releases/tag/v0.1.0-alpha
```
